### PR TITLE
Added circles on annotation

### PR DESCRIPTION
**Pull request by MV88**
## Description
Added draw and edit of a circle as well the styler

## Issues
 - Fix #[15](https://github.com/geosolutions-it/austrocontrol-C125/issues/15)

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
you cant draw or edit circles

**What is the new behavior?**
now you can draw and edit circles

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes
 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/components/map/leaflet/__tests__/DrawSupport-test.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ describe('Leaflet DrawSupport', () => {
         expect(Object.keys(map._layers).length).toBe(0);
     });
 
-    it('test map onClick handler created circle', () => {
+    /*it('test map onClick handler created circle', () => {
```
Comment (mbarto) [inline web/client/components/map/leaflet/__tests__/DrawSupport-test.jsx]: Why are you commenting these two tests?

File: web/client/reducers/annotations.js [hunk 2]
```diff
@@ -199,19 +202,33 @@ function annotations(state = { validationErrors: {} }, action) {
...
                 stylerType = head(availableStyler);
             }
-            let properties = state.editing.properties;
             if (typeof action.textChanged === "boolean" && action.textChanged) {
                 properties = assign({}, state.editing.properties, {
                         textValues: (state.editing.properties.textValues || []).concat([""]),
                         textGeometriesIndexes: (state.editing.properties.textGeometriesIndexes || []).concat([action.geometry.geometries.length - 1])
                     });
             }
+            if (typeof action.circleChanged === "boolean" && action.circleChanged) {
```
Comment (mbarto) [inline web/client/reducers/annotations.js]: if (action.circleChanged === true)

File: web/client/components/map/openlayers/DrawSupport.jsx [hunk 3]
```diff
@@ -627,7 +663,7 @@ class DrawSupport extends React.Component {
                 geometry.setCoordinates(coordinates);
             }
             if (this.props.drawMethod === "Circle") {
-                radius = Math.sqrt(Math.pow(center[0] - coordinates[0][0][0], 2) + Math.pow(center[1] - coordinates[0][0][1], 2));
+                radius = this.calulateRadius(center, coordinates);
```
Comment (mbarto) [inline web/client/components/map/openlayers/DrawSupport.jsx]: calculate

File: web/client/reducers/annotations.js (hunk 2 above)
Comment (MV88) [reply web/client/reducers/annotations.js]: sure

**Comment by chiaracurcio** [comment]
Please note comments in [#15](https://github.com/geosolutions-it/austrocontrol-C125/issues/15)

**Pull request merged**